### ISSUE TITLE
Add groups/heading for the commonly used queries 

### ISSUE THREAD

**Issue by kimrutherford** [pombase/website#2216]
Perhaps we could think about adding sub-heading in this list?  It's getting a bit long.

Some ideas for headings:
  - Gene lists
  - Orthologs/paralogs
  - Unstudied/understudied genes
  - Other

![image](https://github.com/user-attachments/assets/cfa3f7c4-e2ad-4cb2-aaca-d1f292cb7a7a)

**Comment by ValWood** [comment]
I agree.

Definitely 
"orthologs and paralogs"
"Unstudied/understudied genes"

Can we think of anything else for "gene lists", (they are all gene lists)

**Comment by ValWood** [comment]
Maybe "product type" because thats what we use in the full menu

**Comment by kimrutherford** [comment]
Are there any queries we can remove to save vertical space?

**Comment by kimrutherford** [comment]
Yep, I think that looks better.  The change is on the main site.

https://www.pombase.org/query

<img width="1160" height="289" alt="Image" src="https://github.com/user-attachments/assets/f2c4d6fe-b3f8-468b-b7b3-19634f416cdf" />
